packages/sanity/src/_internal/cli/actions/deploy/__tests__/undeployAction.test.ts

```diff
@@ -107,6 +107,32 @@ describe('undeployStudioAction', () => {
     expect(helpers.deleteUserApplication).not.toHaveBeenCalled()
   })
 
+  it(`if running in unattended mode, it doesn't prompt the user for confirmation`, async () => {
+    helpers.getUserApplication.mockResolvedValueOnce(mockApplication)
+    helpers.deleteUserApplication.mockResolvedValueOnce(undefined)
+    ;(mockContext.prompt.single as Mock<typeof mockContext.prompt.single>).mockResolvedValueOnce(
+      true,
+    ) // User confirms
+
+    await undeployStudioAction(
+      {extOptions: {yes: true}} as CliCommandArguments<UndeployStudioActionFlags>,
```

Comment:
Think you need the same thing in a few of the other failing tests in this suite

Reply:
Oh good call––fixed now I believe. I didn't have access to the environment variables to run these locally